corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -135,7 +135,7 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
         <td class="module-case-list-column"><img class="module-icon" src="<%- resolveUri(datum) %>"/></td>
       <% } else if (styles[index].displayFormat === 'ClickableIcon') { %>
         <td class="module-case-list-column">
-          <img class="module-icon btn" src="<%- resolveUri(datum) %>"/>
+          <img class="module-icon" src="<%- resolveUri(datum) %>"/>
```

Comment:
does it need something to distinguish it from other icons that have the same class `module-icon`?
also need change it in the view: https://github.com/dimagi/commcare-hq/blob/db4127aad84631b346d5cbba56520ee96b8d724a/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js#L233